x-pack/plugins/security/public/management/users/users_grid/users_grid_page.tsx

```diff
@@ -110,6 +112,18 @@ export class UsersGridPage extends Component<Props, State> {
           </EuiLink>
         ),
       },
+      {
```

Comment:
You've moved the username to be first, which makes sense because full name is optional. However, the table is by default sorted by "full name". Can we change the default sort field https://github.com/elastic/kibana/blob/58a25faaad88409fcda649dbb204c37d4b0a360b/x-pack/plugins/security/public/management/users/users_grid/users_grid_page.tsx#L197 ?

Reply:
Great catch, for sure!